### ISSUE TITLE
Markdown unordered lists do not work in update notes

### ISSUE THREAD

**Issue by robert-scheck** [fedora-infra/bodhi#265]
Filling the update notes under https://bodhi.fedoraproject.org/updates/new with unordered lists in markdown syntax (https://help.github.com/articles/markdown-basics/) does not work, neither in the preview nor after submitting, visit e.g. https://bodhi.fedoraproject.org/updates/phpMyAdmin-4.4.14-1.fc23 with Firefox 38 ESR from RHEL/CentOS 6. It seems to work properly with a text browser such as w3m (CSS issue?).

**Comment by ralphbean** [comment]
They do work, it just requires that you put an extra newline in front of the list like this:

* this
* is
* a
* list

**Comment by robert-scheck** [comment]
No, they do not work. And this seems to be a CSS issue.

**Comment by ralphbean** [comment]
I just tested with a comment on that update.  To be clear, you have to put an extra newline before the list, like this:

```
Some paragraph text goes here

* this
* is
* a
* list
```

**Comment by robert-scheck** [comment]
![unordered-lists](https://cloud.githubusercontent.com/assets/10982488/9393886/1e2afa18-4785-11e5-9ee1-e1f5e37c7f6a.png)

**Comment by ralphbean** [comment]
Oh, you want the bullets to show up?

**Comment by robert-scheck** [comment]
Yes, well, would be nice. This why I assumed a CSS issue (last sentence of my initial description) ;-)

**Comment by ralphbean** [comment]
Noting here that we'll need to surround each markdown area with a ``<span class="markdown">...</span>`` tag so that we can zero-in the CSS rules with something like:

```css
.markdown {
   list-style-type: cjk-ideographic;
}
```
